### DISCUSSION TITLE
Does Next JS 14 latest version no longer cache data fetched in server component?

### DISCUSSION THREAD

**Discussion by neos04** [vercel/next.js#66615] [Help] [answered]
### Summary

In a previous project, fetched data use to be cached, and after a successful form submission i'll use revalidatePath(), after updating some npm packages i think, now everytime i refresh the page it refetches the data, and same for this new project am working one.

I saw that server Actions were not cached in the docs, so i moved the same function into my server component and no change. I'm using PocketBase as my backend and when i add the option `cache: "force-cache"` to the DB request the data is cached but revalidatePath has no effect... what can i do?
Not to mention, revalidatePath sometimes refreshes my page, more so when i do a page refresh then perform the function that updates data to my database.

**Comment by icyJoseph** [comment]
Are these POST requests? or GET request?

**Reply by neos04** [reply]
GET

**Reply by icyJoseph** [reply]
Can you share an example of how the request is made?

**Reply by neos04** [reply]
```
async function fetchAvailabiltyData() {
  const pb = await initPocketBase();
  const userID = await getUserID();

  try {
    const record = await pb
      .collection("availability")
      .getFirstListItem<FormValues>(`userID="${userID}"`);
    return record;
  } catch (error: any) {
    if (error.status === 404) {
      // Handle 404 not found error
      //   console.log("Record not found.");
      //   console.log(error.message);
      return null;
    } else {
      // Handle other errors
      console.error("Failed to fetch data from PocketBase:", error);
      return null;
    }
  }
}

```
Could it be because i initialized a new pocketbase (SSR) instance which uses cookies?

**Reply by icyJoseph** [reply, marked as answer]
Yes, I guess, we can presume that you are deriving the user identity either from headers or cookies, and that would make the requests opt-out from caching:

> The fetch request uses Authorization or Cookie headers and there's an uncached request above it in the component tree.

**Reply by neos04** [reply]
Ohh, okay.
